app/scss/resultPage.scss

```diff
@@ -1,27 +1,17 @@
 @import "imports/variables";
-
-html {
-  font-size: 62.5%;
-  font-family: asap, sans-serif;
-}
-
-body {
-  background-color: $main-background-color;
-  font-size: 14px;
-  font-size: 1.4rem;
-}
```

Comment:
I assume these are now in the styles doc

Reply:
yeah =]